app/controllers/Controller.scala

```diff
@@ -19,29 +19,56 @@ package controllers
 import config.ViewConfig
 import connectors.des.DesConnector
 import javax.inject.{Inject, Singleton}
+import model.des.{ApprovedInformation, Transaction}
 import play.api.Logger
 import play.api.i18n.Messages
 import play.api.mvc._
+import service.des.DesService
 import uk.gov.hmrc.auth.core.{AuthorisationException, AuthorisedFunctions}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class Controller @Inject() (mcc: MessagesControllerComponents, implicit val viewConfig: ViewConfig, page1: views.html.page1,
+class Controller @Inject() (mcc: MessagesControllerComponents, implicit val viewConfig: ViewConfig, one_payment: views.html.one_payment,
```

Comment:
Messages components can be problematic later because they bring some implicits which can't be overriden. I found that better approach is to use just `ControllerComponents` and have `RequestSupport` in whenever it's needed.
Please see and implement `RequestSupport` in `pay-frontend` project